src/dropdown/dropdown.js

```diff
@@ -61,33 +88,42 @@ export default class Dropdown extends Controller {
 }
 
 /**
- * The basic dropdown model interface.
+ * The basic Dropdown model interface.
  *
  * @memberOf ui.dropdown
- * @interface DropdownModel
+ * @interface ui.dropdown.DropdownModel
  */
 
 /**
- * The label of the dropdown.
+ * The label of the Dropdown.
  *
  * @member {String} ui.dropdown.DropdownModel#label
  */
 
 /**
- * Indicated whether the dropdown is "active", which means that the box with options is visible.
- *
- * @member {Boolean} ui.dropdown.DropdownModel#isOn
- */
-
-/**
- * Whether the dropdown is enabled (can be clicked).
+ * Controls whether the Dropdown is enabled (can be clicked).
  *
  * @member {Boolean} ui.dropdown.DropdownModel#isEnabled
  */
 
 /**
  * Fired when the {@link ui.dropdown.Dropdown#button} is clicked.
+ *
  * See {@link ui.button.ButtonModel#execute}.
  *
  * @event ui.dropdown.DropdownModel#execute
  */
+
+/**
+ * The shared Dropdown model interface.
```

Comment:
That's what I pointed out in the other comment – why is Dropdown uppercase and shared lowercase? It's confusing.